content/docs/api-reference/open.md

```diff
@@ -127,7 +127,7 @@ typically faster than loading the whole data into memory.
 
 ## Example: Accessing private repos
 
-This is just a matter of using the right `repo` argument, for example an SSH URL
+The key for this is to use the right `repo` argument, for example an SSH URL
```

Comment:
What if the person doesn't know what is an SSH key?

Reply:
That's beyond the scope of that doc so it's OK. It's also beyond the scope of this PR. Resolving, thanks.